src/sentry/lang/native/unreal.py

```diff
@@ -16,6 +16,32 @@
 ''')
 
 
+def is_unreal_event(data):
+    """Whether this event is an Unreal crash that should be processed in
+    enhancers. For the legacy codepath this still returns False."""
+    return get_path(data, 'contexts', 'unreal', 'type') == 'unreal'
+
+
+def reprocess_unreal_crash(data):
```

Comment:
This does not "re"-process. This processes.

Reply:
There already exists a `process_unreal_crash` since we still do a lot of things in the endpoint.